src/main/client/scene/game-scene.ts

```diff
@@ -162,6 +164,9 @@ export class GameScene extends Scene {
       this.cameras.main.setRotation(-this.character.rotation);
       this.character.update();
       this.updateOtherPlayers();
+      if (this.updateMapSprite) {
+         this.mapSprite?.update();
```

Comment:
probably dont need the `?.` here as its checked above

Reply:
Thanks